src/components/FiatOnrampAnnouncement/index.tsx

```diff
@@ -113,6 +114,7 @@ export function FiatOnrampAnnouncement() {
 
   const toggleWalletDropdown = useToggleWalletDropdown()
   const handleClick = useCallback(() => {
+    sendAnalyticsEvent('FOR Banner Click')
```

Comment:
These analytics events should be encoded as an enum through https://github.com/Uniswap/analytics-events I think

Reply:
Yeah, now that the feature is launched I'm planning on doing that. (We couldn't before because it was a public repo.) I'd like to get this in as is though, and do all the onramp events as one PR into the analytics repo.